### DISCUSSION TITLE
How do I set the blockage to HARD?

### DISCUSSION THREAD

**Discussion by huang-kaixin** [The-OpenROAD-Project/OpenROAD-flow-scripts#1465] [Q&A] [answered]
![微信图片_20230914173806](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/assets/111551571/e088dbc1-351c-4599-9e8c-b527cf579f53)
I want to set the blockage around the macro to hard, prohibiting standard cells from being placed in this area. However, I found that the blockage only has the setSoft method. I tried deleting the $b setSoft command in the placement_blockages.tcl, setting soft to false, but standard cells can still be placed in the blockage. Is my approach incorrect? If so, is there a method to set the blockage around the macro to hard and prevent standard cells from being placed in this area?

**Comment by rovinski** [comment, marked as answer]
Answered in #1466
